### PR TITLE
CIV-14389 Upload directions order docs

### PR DESCRIPTION
**Pull request by paul-pearson**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-14389

### Change description ###

Implement directions order doc upload

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/common/models/generalApplication/GeneralApplication.ts [hunk 1]
```diff
@@ -56,7 +58,8 @@ export class GeneralApplication {
     uploadN245Form?: UploadGAFiles,
     informOtherParties?: InformOtherParties,
     applicationFee?: ClaimFeeData,
-  ) {
+    generalAppAddlnInfoUpload?: UploadGAFiles,
```
Comment (neeta-hmcts) [inline src/main/common/models/generalApplication/GeneralApplication.ts:61]: don't need these fields.. can be removed. As using draftStore to save doc

File: src/main/services/features/generalApplication/generalApplicationService.ts [hunk 2]
```diff
@@ -385,6 +386,18 @@ export const saveRespondentWantToUploadDoc = async (claimId: string, claim: Clai
   }
 };
 
+export const getClaimDetailsById = async (req: AppRequest): Promise<Claim> => {
```
Comment (neeta-hmcts) [inline src/main/services/features/generalApplication/generalApplicationService.ts:389]: Can be removed this method. as its not being used  ( it came from my code.. I missed to remove if )

File: src/main/common/models/generalApplication/GeneralApplication.ts (hunk 1 above)
Comment (paul-pearson) [reply src/main/common/models/generalApplication/GeneralApplication.ts]: Fields removed

File: src/main/services/features/generalApplication/generalApplicationService.ts (hunk 2 above)
Comment (paul-pearson) [reply src/main/services/features/generalApplication/generalApplicationService.ts]: Function removed

**Review by neeta-hmcts — DISMISSED** [review]

**Review by jeswanth-hmcts — DISMISSED** [review]

**Pull request merged**